### PR TITLE
Fix #2198: Document encoded=True option

### LINKED ISSUE #2198: Document encoded=True for yarl.URL
See also #2188

### PR DESCRIPTION
**Pull request by asvetlov**
Fix for #2198 

Guys, I not sure about wording. Please feel free to propose better text.

### CODE REVIEW

File: docs/client.rst [hunk 1]
```diff
@@ -106,6 +106,25 @@ is not encoded by library. Note that ``+`` is not encoded::
...
+   ``URL('http://example.com/%D0%BF%D1%83%D1%82%D1%8C/0?a=1')``.
+
+   Sometimes canonization is not desirable if server accepts exact
+   representation and doesn't requote URL itself.
+
+   To disable canonization use ``encoded=True`` parameter for URL construction::
+
+      await session.get(URL('http://example.com/%30', encoded=True))
+
+   Passing *params* overrides ``encoded=True``, never use both options.
```
Comment (kxepal) [inline docs/client.rst]: On second thought, may be put that line into `.. warning` block? Would be hard to pass by.

Comment (asvetlov) [reply docs/client.rst]: Put warning next after note section?

Comment (kxepal) [reply docs/client.rst]: Yes.

File: docs/glossary.rst [hunk 2]
```diff
@@ -52,6 +52,18 @@
 
        http://gunicorn.org/
 
+   IDNA
+
+       An Internationalized Domain Name in Applications (IDNA) is an
+       standard for encoding Internet Domain Names that contain in
```
Comment (samuelcolvin) [inline docs/glossary.rst]: "is **a** standard"

Comment (samuelcolvin) [reply docs/glossary.rst]: or maybe "is an industry standard"

Comment (asvetlov) [reply docs/glossary.rst]: Fixed.

@samuelcolvin I'm not a native English speaker.
If you feel that documentation could be improved and have time for the job -- please don't hesitate.

Comment (samuelcolvin) [reply docs/glossary.rst]: > and have time

:smile: 

I think overall it's pretty good. If I see anything jarring I'll try to correct.

**Pull request merged**